.github/workflows/services-extract-tag.yml

```diff
@@ -11,7 +11,6 @@ on:
 jobs:
   extract-service-tag:
     runs-on: ubuntu-latest
-    if: ${{ github.actor != 'github-actions[bot]' && github.event_name != 'push' || github.event.pusher.name != 'github-actions[bot]' }}
```

Comment:
Made this workflow simpler: we don't care about whether we are on the master branch or not, it's up to the caller to care about it.